Cargo.toml

```diff
@@ -15,8 +15,15 @@ memmap = "0.7"
 farmhash = "1.1"
 prost = "0.6"
 
+[dev-dependencies]
+criterion = "0.3"
+
 [workspace]
 members = [
     "proto",
     "skiplist",
 ]
+
+[[bench]]
```

Comment:
Should add a new profile for bench. `codegen-units` should be set to 1 to get a correct bench result.

Reply:
I copied TiKV `[profile.bench]` to `Cargo.toml`